src/main/resources/com/google/api/codegen/go/sample.snip

```diff
@@ -155,12 +164,42 @@
   }
 @end
 
-@private callChain(methodNameComponents)
-  @join resource : methodNameComponents on "."
+@private field(field)
+  @if field.description
+    @join line : util.getDocLines(field.description, 100)
+      // {@line}
+    @end
+
+
+  @end
+  {@field.name} := {@field.defaultValue}  // TODO: Update placeholder value.
+  @if field.example
+    // {@field.example}
+  @end
+@end
+
+@private callChain(class)
```

Comment:
Can also include `{@class.ServiceVarName}` in this snippet. I'd prefer to call it something like `methodCall`.